### ISSUE TITLE
Question regarding learning rate.

### ISSUE THREAD

**Issue by PratsBhatt** [amaiya/ktrain#163]
Hello Community,

I find this project very inspiring and have started using this project for my needs.
It was really easy to do transfer learning using ktrain on Hugging face models.

I do have a question though, I see that there is an api that finds the best learning rate for training the model like 

`learner.lr_find()             # briefly simulate training to find good learning rate
learner.lr_plot()             # visually identify best learning rate`

But, in all the examples, when training the model the learning rate is always hardcoded, I didn't find any tutorial that uses the best learning rate found using the `lr_find()` api.
`# train using 1cycle learning rate schedule for 3 epochs
learner.fit_onecycle(2e-5, 3) `
Could someone direct me, or is my understanding wrong?

Thank you in advance for your help and guidance.

**Comment by amaiya** [comment]
Hello @PratsBhatt  Your question is actually answered exactly in this [Medium tutorial](https://medium.com/@asmaiya/text-classification-with-hugging-face-transformers-in-tensorflow-2-without-tears-ee50e4f3e7ed) and corresponding [tutorial notebook](https://nbviewer.jupyter.org/github/amaiya/ktrain/blob/master/tutorials/tutorial-A3-hugging_face_transformers.ipynb), which are both listed in the README page of *ktrain*.

In these tutorials, the learning-rate-finder step is listed as optional because learning rates between `2e-5` and `5e-5` typically work well with `transformer-based` models like BERT.  So, if you just use a learning rate of say `3e-5`, things will typically be fine.  That said, in the Mediium article and tutorial notebook above, the learning-rate-finder is executed anyways and ends up indicating a recommended learning rate within this range, as further evidence it is optional for `transformer` models. like BERT and DistilBERT.
